### PR TITLE
Remove deprecated workspace collection cmdlets

### PR DESCRIPTION
**Pull request by ayeshurun**
## Checklist

- [X] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/PowerBIEmbedded/PowerBIEmbedded/Az.PowerBIEmbedded.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.2.1'
+ModuleVersion = '2.0.0'
```
Comment (VeryEarly) [inline src/PowerBIEmbedded/PowerBIEmbedded/Az.PowerBIEmbedded.psd1:15]: revert this line

File: src/PowerBIEmbedded/PowerBIEmbedded/ChangeLog.md [hunk 2]
```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 2.0.0
```
Comment (VeryEarly) [inline src/PowerBIEmbedded/PowerBIEmbedded/ChangeLog.md:22]: move line 23 to line 21 and remove line 22

File: src/PowerBIEmbedded/PowerBIEmbedded.Test/PowerBIEmbedded.Test.csproj [hunk 3]
```diff
@@ -14,4 +14,8 @@
     <PackageReference Include="Microsoft.Azure.Management.PowerBIEmbedded" Version="1.1.1-preview" />
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (VeryEarly) [inline src/PowerBIEmbedded/PowerBIEmbedded.Test/PowerBIEmbedded.Test.csproj:17]: please remove this section

**Review by VeryEarly — CHANGES_REQUESTED** [review]

**Comment by ayeshurun** [comment]
@VeryEarly resolved all comments. would appreciate a review

File: src/PowerBIEmbedded/PowerBIEmbedded/Az.PowerBIEmbedded.psd1 (hunk 1 above)
Comment (VeryEarly) [reply src/PowerBIEmbedded/PowerBIEmbedded/Az.PowerBIEmbedded.psd1:15]: please do revert this line

**Comment by VeryEarly** [comment]
> @VeryEarly resolved all comments. would appreciate a review

please revert version change in [src/PowerBIEmbedded/PowerBIEmbedded/Az.PowerBIEmbedded.psd1](https://github.com/Azure/azure-powershell/pull/23099/files/9ba9199a6ba66dbead418688b670c39f3f65799b#diff-7b23a994a2b6de33fc11d52d16d19aa72df9743b8edbc80861b546fffe081a75)

**Comment by ayeshurun** [comment]
@VeryEarly resolved comments. hopefully last iteration. please review :)

**Comment by ayeshurun** [comment]
@a1exwang / @VeryEarly would appreciate your feedback.

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**